docs/academy/blockchain-basics/transaction-basics.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Sending Transactions
+slug: /transactions-basics
```

Comment:
The slug should match the file name as well, removing the `s` from `transactions`